packages/Webkul/Shop/src/Resources/views/layouts/footer/footer.blade.php

```diff
@@ -7,7 +7,7 @@
 
                 foreach (app('Webkul\Category\Repositories\CategoryRepository')->getVisibleCategoryTree(core()->getCurrentChannel()->root_category_id) as $category){
                     if ($category->slug)
-                        array_push($categories, $category);
+                        $categories[] = $category;
```

Comment:
- cleaner; 
- faster;

[more here](https://stackoverflow.com/questions/559844/which-is-faster-in-php-array-value-or-array-pusharray-value)